### ISSUE TITLE
'IUE' keyword error

### ISSUE THREAD

**Issue by petiay** [karllark/measure_extinction#84]
When getting the model data, using [AzV23 as an SMC star test](https://github.com/karllark/extstar_data/blob/master/DAT_files/azv23.dat), a KeyError results:

```
~/Documents/ext-science/measure_extinction/modeldata.py in __init__(self, modelfiles, path, band_names, spectra_names)
    117                 # initialize the spectra vectors
    118                 if self.fluxes[cspec] is None:
--> 119                     self.waves[cspec] = moddata.data[cspec].waves
    120                     self.fluxes[cspec] = np.zeros(
    121                         (self.n_models, len(moddata.data[cspec].fluxes))

KeyError: 'IUE'
```
@karllark, should the .dat file say "STIS = STIS_Data..." instead of "IUE = STIS_Data..."?

I only see `spectra_names=["BAND", "STIS"]` defined in [modeldata.py](https://github.com/karllark/measure_extinction/blob/master/measure_extinction/modeldata.py
).

**Comment by karllark** [comment]
Yeah, this is an issue I'm aware of.  I haven't figured out a way around this yet, but UV data from STIS and IUE are currently interchangeable.  So just change the dat files as you indicate "IUE = STIS_Data..." and it will work.  The best solution is to generate both IUE and STIS model spectra.  Will work on that.  Keep this issue open so that we make sure to solve it the best way.

**Comment by petiay** [comment]
Sounds good - this is indeed what I ended up doing and it seemed to work.

**Comment by karllark** [comment]
This has been fixed by having both IUE and STIS data properly mocked from the model spectra.   If this issue is till relevant, please reopen.
